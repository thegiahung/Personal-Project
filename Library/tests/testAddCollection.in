Add collection books.csv
Add collection books.csv
Add book books.csv 111111
Add collection hello.csv
Exit
