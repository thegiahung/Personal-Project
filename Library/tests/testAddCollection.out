user: 17 books successfully added.

user: No books have been added to the system.

user: Book already exists in system.

user: No such collection.

user: Ending Library process.
